lib/model.js

```diff
@@ -407,15 +408,19 @@ assign(derived, {
     fn() {
       const AUTH_TOKEN = 'AUTH_TOKEN';
       const req = {
-        protocol: this.isSrvRecord ? 'mongodb+srv' : 'mongodb',
+        protocol: 'mongodb',
```

Comment:
suggestion: might be worth adding a few comments around this code related to the bug this PR fixes.

Reply:
this particular line doesn't affect functionality. i just moved the logic of protocol setting to the `if` statement below in code:
```
if (this.isSrvRecord) {
  req.protocol = 'mongodb+srv';
  ...
}
``` 
but i will add comments about srv edge cases. you are right, it will be helpful in the future.

Reply:
Added some comments about srv. Thanks for bring this up!